### PR TITLE
Making changes to Presenter Information

### PR DESCRIPTION
**Pull request by AfiMaameDufie**
## Description

* Added an image to replace the dark blank image 
* Took out the initial image
* Summarized the content in the presenter info section

CC: @tim-schilling

### CODE REVIEW

**Review by tim-schilling — CHANGES_REQUESTED** [review]
There's a small issue with the photo_url in your markdown file. If you drop one of the .jpeg segments, this will be good to go.

File: _presenters/abigail-afi-gbadago.md [hunk 1]
```diff
@@ -5,20 +5,13 @@ layout: speaker-template
 mastodon: null
 name: Abigail Afi Gbadago
 permalink: /presenters/abigail-afi-gbadago/
-photo_url: /static/img/presenters/abigail-afi-gbadago.jpeg
+photo_url: /static/img/presenters/abigail-gbadago.jpeg.jpeg
```
[the remark this answers is not in the document]
Comment (AfiMaameDufie) [reply _presenters/abigail-afi-gbadago.md:8]: Oww, typo. Will change that now. Thank you.

**Comment by AfiMaameDufie** [comment]
> There's a small issue with the photo_url in your markdown file. If you drop one of the .jpeg segments, this will be good to go.

Fixed

**Review by tim-schilling — APPROVED** [review]
Excellent, thank you!

**Pull request merged**